app/move/controllers/create/base.test.js

```diff
@@ -0,0 +1,127 @@
+const FormController = require('hmpo-form-wizard').Controller
+
+const CreateBaseController = require('./base')
+
+const controller = new CreateBaseController({ route: '/' })
+
+describe('Move controllers', function() {
+  describe('Create base controller', function() {
+    describe('#middlewareChecks()', function() {
+      beforeEach(function() {
+        sinon.stub(FormController.prototype, 'middlewareChecks')
+        sinon.stub(controller, 'use')
+
+        controller.middlewareChecks()
+      })
+
+      it('should call parent method', function() {
+        expect(FormController.prototype.middlewareChecks).to.have.been
+          .calledOnce
+      })
+
+      it('should call check current location method', function() {
+        expect(controller.use).to.have.been.calledWith(
+          controller.checkCurrentLocation
+        )
+      })
+    })
+
+    describe('#middlewareLocals()', function() {
+      beforeEach(function() {
+        sinon.stub(FormController.prototype, 'middlewareLocals')
+        sinon.stub(controller, 'use')
+
+        controller.middlewareLocals()
+      })
+
+      it('should call parent method', function() {
+        expect(FormController.prototype.middlewareLocals).to.have.been
+          .calledOnce
+      })
+
+      it('should call set cancel url method', function() {
+        expect(controller.use).to.have.been.calledWith(controller.setCancelUrl)
+      })
+    })
+
+    describe('#setCancelUrl()', function() {
+      let res, nextSpy
+
+      beforeEach(function() {
+        nextSpy = sinon.spy()
+        res = {
+          locals: {},
+        }
+      })
+
+      context('with no moves url local', function() {
+        beforeEach(function() {
+          controller.setCancelUrl({}, res, nextSpy)
+        })
+
+        it('should set cancel url correctly', function() {
+          expect(res.locals).to.have.property('cancelUrl')
+          expect(res.locals.cancelUrl).to.equal(undefined)
```

Comment:
more personal preference but you could aslo use 
```
.to.be.an('undefined')
```

Reply:
Yeah, that's much nicer. Again, may do these kind of tweaks as a separate change to not confuse with what this change is doing.

Reply:
Have added #208 to amend this across the codebase.